v3/src/components/graph/components/inspector-panel/display-config-palette.tsx

```diff
@@ -62,12 +64,20 @@ export const DisplayConfigPalette = observer(function DisplayConfigPanel(props:
   }
 
   const handleSetFuseIntoBars = (fuseIntoBars: boolean) => {
-    graphModel?.setPointsFusedIntoBars(fuseIntoBars)
-    if (fuseIntoBars) {
-      graphModel?.pointDescription.setPointStrokeSameAsFill(true)
-    } else {
-      graphModel?.pointDescription.setPointStrokeSameAsFill(false)
-    }
+    const [undoStringKey, redoStringKey] = fuseIntoBars
+      ? ["DG.Undo.graph.fuseDotsToRectangles", "DG.Redo.graph.fuseDotsToRectangles"]
+      : ["DG.Undo.graph.dissolveRectanglesToDots", "DG.Redo.graph.dissolveRectanglesToDots"]
+
+    graphModel?.applyModelChange(() => {
```

Comment:
While testing things I realized I had left out adding undo/redo support for this handler. I tried to likewise add it for `handleSelection` above, but ran into some weird problems and decided to skip that for this PR.